resources/istio/charts/kyma/templates/label-job.yaml

```diff
@@ -0,0 +1,50 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: kyma-ns-label
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: kyma-ns-label
+rules:
+- apiGroups: [""]
+  resources: ["namespaces"]
+  verbs: ["get", "list", "watch", "patch"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: kyma-ns-label
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: kyma-ns-label
+subjects:
+- kind: ServiceAccount
+  name: kyma-ns-label
+  namespace: istio-system
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
```

Comment:
I would invoke this as a post install & upgrade step and add some logging there when it fails